test/cql-pytest/test_uda.py

```diff
@@ -107,13 +107,13 @@ def test_incorrect_state_func(scylla_only, cql, test_keyspace):
     div_body = "(state tuple<bigint, bigint>) CALLED ON NULL INPUT RETURNS bigint LANGUAGE lua AS 'return state[1]//state[2]'"
     with new_function(cql, test_keyspace, avg_partial_body) as avg_partial, new_function(cql, test_keyspace, div_body) as div_fun:
         custom_avg_body = f"(bigint) SFUNC {avg_partial} STYPE tuple<bigint, bigint> FINALFUNC {div_fun} INITCOND (0,0)"
-        with pytest.raises(InvalidRequest, match="State function not found"):
+        with pytest.raises(InvalidRequest, match=r"State function (.*) not found"):
```

Comment:
Nitpick: you don't need the parentheses in the regex - they don't do anything.
Also, why did you add the "r"? What is it supposed to achieve here?

Reply:
I've added the prefix by mistake, removed.